changelog.d/8196.misc

```diff
@@ -0,0 +1 @@
+Fix `wait_for_stream_position` to allow multiple waiters on same stream ID.
```

Comment:
This isn't a bugfix as this can't currently happen, it only starts being a problem with sharded event persister.

I also plan to squash all these changelogs together anyway.